org.gnome.Radio.json

```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
+  ],
+    "modules": [
+	{
+	    "name": "intltool",
+	    "cleanup": [ "*" ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+		    "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+		}
+	    ]
+	},
+	{
+            "name": "cogl",
+            "config-opts": [
+		"--disable-cogl-gst",
+		"--disable-gtk-doc",
+		"--enable-xlib-egl-platform",
+		"--enable-wayland-egl-platform"
+            ],
+            "cleanup": [
+		"/share/gtk-doc",
+		"/share/cogl/examples-data"
+            ],
+            "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/cogl/1.22/cogl-1.22.8.tar.xz",
+                    "sha256": "a805b2b019184710ff53d0496f9f0ce6dcca420c141a0f4f6fcc02131581d759"
+		}
+            ]
+	},  
+	{
+            "name": "clutter",
```

Comment:
please use clutter from the shared-modules.

Reply:
Fixed